[2024-12-04T02:42:31.697+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-04T02:42:31.763+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: complete-dag.fetch_forecast_weather scheduled__2024-12-02T20:40:00+00:00 [queued]>
[2024-12-04T02:42:31.771+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: complete-dag.fetch_forecast_weather scheduled__2024-12-02T20:40:00+00:00 [queued]>
[2024-12-04T02:42:31.771+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-04T02:42:31.790+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_forecast_weather> on 2024-12-02 20:40:00+00:00
[2024-12-04T02:42:31.841+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=240) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-04T02:42:31.832+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'complete-dag', 'fetch_forecast_weather', 'scheduled__2024-12-02T20:40:00+00:00', '--job-id', '465', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmpdegcgv3j']
[2024-12-04T02:42:31.843+0000] {standard_task_runner.py:63} INFO - Started process 249 to run task
[2024-12-04T02:42:31.843+0000] {standard_task_runner.py:91} INFO - Job 465: Subtask fetch_forecast_weather
[2024-12-04T02:42:32.054+0000] {task_command.py:426} INFO - Running <TaskInstance: complete-dag.fetch_forecast_weather scheduled__2024-12-02T20:40:00+00:00 [running]> on host e060654747dc
[2024-12-04T02:42:32.329+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='complete-dag' AIRFLOW_CTX_TASK_ID='fetch_forecast_weather' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T20:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-02T20:40:00+00:00'
[2024-12-04T02:42:32.332+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-04T02:42:37.869+0000] {python.py:237} INFO - Done. Returned value was:          Date  temperature_2m_max  ...  windspeed_10m_max  avg_temperature
0  2024-12-03                 8.9  ...                3.7             3.00
1  2024-12-04                 8.5  ...               18.5             4.00
2  2024-12-05                 7.6  ...               14.4             2.85
3  2024-12-06                 5.8  ...               19.1             1.25
4  2024-12-07                 5.7  ...               15.9             0.55
5  2024-12-08                 7.0  ...                9.7             0.90
6  2024-12-09                 7.3  ...               18.4             2.50

[7 rows x 7 columns]
[2024-12-04T02:42:37.952+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-04T02:42:38.177+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=complete-dag, task_id=fetch_forecast_weather, run_id=scheduled__2024-12-02T20:40:00+00:00, execution_date=20241202T204000, start_date=20241204T024231, end_date=20241204T024238
[2024-12-04T02:42:38.260+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-04T02:42:38.325+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
